lineman/spec-e2e/proposal_spec.coffee

```diff
@@ -89,3 +89,30 @@ describe 'Proposals', ->
       emailHelper.openLastEmail()
       emailHelper.clickAgreeLink()
       expect(proposalsHelper.voteFormPositionSelect().getAttribute('value')).toContain('yes')
+
+  describe 'undecided members', ->
+
+      describe 'when proposal is open', ->
+
+        beforeEach ->
+          threadHelper.loadWithActiveProposalWithVotes()
+          proposalsHelper.clickShowUndecidedLink()
+
+        it 'shows all undecided members when the show link is clicked', ->
+          expect(proposalsHelper.undecidedPanel().getText()).toContain('Emilio')
+          expect(proposalsHelper.undecidedPanel().getText()).toContain('Hide undecided members')
+
+        it 'hides all undecided members when undecided panel is open and hide link is clicked', ->
+          proposalsHelper.clickHideUndecidedLink()
+          expect(proposalsHelper.undecidedPanel().getText()).toContain('Show undecided members')
+
+      describe 'when proposal is closed', ->
+
+        beforeEach ->
+          threadHelper.loadWithClosedProposal()
+
+        xit 'shows all undecided members when the show link is clicked', ->
```

Comment:
Hey James - it is working but it's a little complicated to test - you need a closed proposal, then a new member added, and I couldn't get the tests to work so I'm gonna have another stab again soon